src/main/java/teammates/ui/controller/StudentProfilePageData.java

```diff
@@ -1,14 +1,45 @@
 package teammates.ui.controller;
 
 import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.datatransfer.StudentProfileAttributes;
+import teammates.common.util.Const;
+import teammates.ui.template.StudentProfileEditBox;
+import teammates.ui.template.StudentProfileUploadPhotoModal;
 
 public class StudentProfilePageData extends PageData {
 
-    public String editPicture;
-
+    private StudentProfileEditBox profileEditBox;
+    private StudentProfileUploadPhotoModal uploadPhotoModal;
+    
     public StudentProfilePageData(AccountAttributes account, String editPicture) {
```

Comment:
You can set the variable name as isEditingPhoto can name the getter method as isEditingPhoto().
Call it from the tag using ${profile.editingPhoto}